DataConnectors/Flare Systems Firework/Connector_REST_API_FlareSystemsFirework.json

```diff
@@ -0,0 +1,97 @@
+            "type": "IsConnectedQuery",
+            "value": [
+                "Firework_CL\n            | summarize LastLogReceived = max(TimeGenerated)\n            | project IsConnected = LastLogReceived > ago(30d)"
+            ]
+        }
+    ],
+    "availability": {
+        "status": 1,
+        "isPreview": true
+    },
+    "permissions": {
+        "resourceProvider": [
+            {
+                "provider": "Microsoft.OperationalInsights/workspaces",
+                "permissionsDisplayText": "read and write permissions are required.",
+                "providerDisplayName": "Workspace",
+                "scope": "Workspace",
+                "requiredPermissions": {
+                    "write": true,
+                    "read": true,
+                    "delete": true
+                }
+            },
+            {
+                "provider": "Microsoft.OperationalInsights/workspaces/sharedKeys",
+                "permissionsDisplayText": "read permissions to shared keys for the workspace are required. [See the documentation to learn more about workspace keys](https://docs.microsoft.com/azure/azure-monitor/platform/agent-windows#obtain-workspace-id-and-key).",
+                "providerDisplayName": "Keys",
+                "scope": "Workspace",
+                "requiredPermissions": {
+                    "action": true
+                }
+            }
+        ],
+        "customs": [
+            {
+                "name": "Required Firework permissions",
+                "description": "only Firework organization administrators may configure the Azure Sentinel integration."
+            }
+        ]
+    },
+    "instructionSteps": [
+        {
+            "title": "1. Creating an Alert Channel for Azure Sentinel",
+            "description": "",
+            "innerSteps": [
+                {
+                    "title": "",
+                    "description": "As an organization administrator, authenticate on [Firework](https://firework.flared.io/) and access the [team page](https://firework.flared.io/#/team) to create a new alert channel."
+                },
+                {
+                    "title": "",
+                    "description": "Click on 'Create a new alert channel' and select 'Azure Sentinel'. Enter your Shared Key And WorkspaceID. Save the Alert Channel."
+                }
+            ]
+        },
+        {
+            "title": "2. Associating your alert channel to an alert feed",
+            "description": "",
+            "innerSteps": [
+                {
+                    "title": "",
+                    "description": "At this point, you may configure alerts to be sent to Azure Sentinel the same way that you would configure regular email alerts."
+                },
+                {
+                    "title": "",
+                    "description": "For a more detailed guide, refer to the Firework documentation."
+                }
+            ]
+        }
+    ]
+}
```

Comment:
Please add a Metadata object within this file. See https://github.com/Azure/Azure-Sentinel/blob/master/DataConnectors/Templates/Connector_REST_API_template.json for further detail.

Reply:
Done! I wasn't sure what to set for the `id`, let me know if I need to change it.